modules/ossm-about-service-mesh-custom-resource-definitions.adoc

```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+// * install/ossm-installing-openshift-service-mesh.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ossm-about-service-mesh-custom-resource-definitions_{context}"]
+= About Service Mesh Custom Resource Definitions
+
+Installing the {SMProductName} Operator also installs Custom Resource Definitions (CRD) that network administrators can use to configure {istio} for {SMProductShortName} installations. There are two different categories of CRDs: {SMProduct} CRDs and {istio} CRDs.
```

Comment:
I know you'll probably want to replace this with a variable, but this is what I see as more accurate.
```suggestion
Installing the {SMProductName} Operator also installs Custom Resource Definitions (CRD) that network administrators can use to configure {istio} for {SMProductShortName} installations. There are two different categories of CRDs: sail-operator CRDs and {istio} CRDs.
```

Reply:
done